### PR TITLE
Matched the pair of lock and unlock

### PR DESCRIPTION
**Pull request by txuna**
I am a user of onion framework better than anyone else.
I decided that matching the lock and unlock pairs is good for users viewing the example/oterm/oterm_handler.c#120

### CODE REVIEW

**Comment by davidmoreno** [comment]
Hi,

I dont quite understand whats the purpose of the commit.

Functionally should do the same. The only difference is one unlock instead of two depending on the if condition.

Can you explain the benefits?

Regards,
David.

**Comment by txuna** [comment]
First, thank you for watching my commit and I not good at english

Of course, functionally and logically, there are no bugs. However, if the number of locks and unlocks is not match, it can lead to bugs in terms of maintenance. And that code is example code, it can be code that can be referenced by many users(about beginner), and it is better to block bugs in advance when users try to extend the code.

In conclusion, there are no bugs, but it is important to keep in mind that when the code is expanded, there may be bugs in terms of maintenance. So, I decided that it would be better to match the pair of lock and unlock.

Regards, 
txuna.
